Source/PCH/pch.h

```diff
@@ -54,6 +60,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <string_view>
```

Comment:
Would it make sure to include `<string>` as well?

Reply:
I was just going by profiler metrics so...probably not ? ( I guess it was not reported in the top hits/time)

Reply:
You (or anyone) can easily profile the msvc build, I recommend starting here: https://devblogs.microsoft.com/cppblog/author/kevca/